src/EnergyPlus/SizingManager.cc

```diff
@@ -1843,6 +1900,11 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
                                                              TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
                                                              state.dataSize->VpzMinClgByZone(termUnitSizingIndex),
                                                              4); // Vpz-min
+                    // Vpz-min, simplified procedure?
+                    if (TermUnitFinalZoneSizing(termUnitSizingIndex).VpzMinByZoneSPSized) {
+                        OutputReportPredefined::PreDefTableEntry(
+                            state, state.dataOutRptPredefined->pdchS62zcdVpzminSPSize, TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName, "Yes");
+                    }
```

Comment:
New output report zone column that reports whether or not the zone minimum primary air flow is calculated using the Standard 62.1 Simplified Procedure.

Reply:
I finished running the CI locally with updates. The results look pretty good. The table and textual differences are from input fields addition and also expected output differences.

One thing I noticed is that the Vpz-min SP or not tabular output only answers "Yes"; and left empty for the "no" answer: 
![image](https://user-images.githubusercontent.com/64040111/130266458-1aba5113-8617-479f-8dae-47e9093f6305.png)


I do not have a preference on whether the empty entry should be filled with "no"--just for your discretion here.